website/src/examples/mapbox/clusters.mdx

```diff
@@ -0,0 +1,5 @@
+# Clusters
+
+import App from 'mapbox-examples/clusters/src/app';
+
+<App />
```

Comment:
Is this compiled into to a running example?

Reply:
Yes, this is the example page